Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -163,6 +164,11 @@ + (NSString *)cellReuseIdentifier
     return NSStringFromClass([self class]);
 }
 
+- (UIFont *)textMessageFont
+{
+    return [UIFont ows_dynamicTypeBodyFont];
```

Comment:
This PR also includes applying dynamic type in two places:

* Conversation text message bubbles.
* Text input area.